mwclient/client.py

```diff
@@ -612,6 +615,156 @@ def get_token(self, type, force=False, title=None):
 
         return self.tokens[type]
 
+    def create_user(self, username, password, **kwargs):
+        if 'createtoken' not in kwargs:
+            kwargs['createtoken'] = self.get_token('createaccount')
+        if 'retype' not in kwargs:
+            kwargs['retype'] = password
+        if 'continue' not in kwargs and 'createreturnurl' not in kwargs:
+            # FIXME should be great if API didn't require this...
+            kwargs['createreturnurl'] = '%s://%s' % (self.scheme, self.host)
+        info = self.post('createaccount', username=username, password=password, **kwargs)
+        if info['createaccount']['status'] == 'FAIL':
+            raise Exception('Unable to createaccount [%s]: %s' % (
+                info['createaccount']['messagecode'], info['createaccount']['message'])
+            )
+        return True
+
+    def get_user(
+        self, username=None, userid=None,
+        usprop='registration|groups|blockinfo'
+    ):
+        if (not username and not userid) or (username and userid):
+            raise Exception('username OR userid are required')
```

Comment:
Please use a subclass instead. [ValueError](https://stackoverflow.com/questions/256222/which-exception-should-i-raise-on-bad-illegal-argument-combinations-in-python) perhaps?

Reply:
done